sql/moz-fx-data-shared-prod/udf/pseudonymize_ad_id/README.md

```diff
@@ -0,0 +1,9 @@
+
+<!--
+Warning: Only for use in the Ad ID pipeline.
+
+This UDF uses a key to pseduonymize Ad IDs,
+and handles the all-0 Ad IDs we get from
+opted-out clients.
+-->
```

Comment:
```suggestion
Warning: Only for use in the Ad ID pipeline.

This UDF uses a key to pseduonymize Ad IDs,
and handles the all-0 Ad IDs we get from
opted-out clients.
```

I don't think it is strictly necessary to have this as a comment. Though these docs aren't being rendered anywhere, so I guess it doesn't really matter